app/controllers/project_settings_controller.rb

```diff
@@ -0,0 +1,99 @@
+#-- encoding: UTF-8
```

Comment:
I am not sure where to put this, so I drop it here. When changing the controller name, like you did here, you need to check for implicit i18n consequences. The "no_results_box" is one of those. This text does not get shown properly anymore for instance:         https://github.com/opf/openproject/blob/aa62568debd71a87697ab6f93422a7192091636c/config/locales/en.yml#L143